tests/selenium/framework/page-objects/MainPage.js

```diff
@@ -57,7 +58,7 @@ class MainPage extends BasePage {
   }
 
   getPricingLinkElement() {
-    return element(by.xpath(pricingLinkXpath));
+    return element(by.css(signupPricingSelector));
```

Comment:
Do we have to use a css selector for this? Why won't the XPath selector work?

Reply:
yes. so selenium test will open a new page with okta.com site and it pricing page also includes `Pricing` inside, so in console, In such case, I see the warning `more than one element found for locator By(xpath, //*[text()='Pricing']) - the first result will be used`. So that's why I decide to use more unique selector

Reply:
Is there a way we can still keep using xpath but use a more specific path expression? I'd like to avoid adding special logic in the footer rendering just to insert the CSS class for a selenium test.